visualization/app/codeCharta/util/nodeDecorator.ts

```diff
@@ -27,7 +28,6 @@ export class NodeDecorator {
 
 	public static preDecorateFile(file: CCFile) {
 		this.decorateMapWithPathAttribute(file)
-		this.decorateNodesWithIds(file.map)
 	}
```

Comment:
Keeping the `preDecorateFile()` function is probably not necessary anymore and we might instead call `decorateMapWithPathAttribute()` directly instead?

Reply:
100% agree